src/extension.ts

```diff
@@ -346,7 +346,10 @@ function launchMetals(
       }
     };
     Object.entries(clientCommands).forEach(([name, command]) =>
-      registerCommand(name, command)
+      registerCommand(
+        ClientCommands[name as keyof typeof ClientCommands],
```

Comment:
this is the fix for ClientCommands: we were using `name` instead of accessing the id.

For example, we were registering the command as `runDoctor` instead of `metals-doctor-run`